homeassistant/components/samsungtv/bridge.py

```diff
@@ -37,20 +41,24 @@
 )
 
 
-def mac_from_device_info(info):
+def mac_from_device_info(info: Any) -> str | None:
     """Extract the mac address from the device info."""
     dev_info = info.get("device", {})
     if dev_info.get("networkType") == "wireless" and dev_info.get("wifiMac"):
         return format_mac(dev_info["wifiMac"])
     return None
 
 
-async def async_get_device_info(hass, bridge, host):
+async def async_get_device_info(
+    hass: HomeAssistant, bridge: SamsungTVBridge | None, host: str
+) -> tuple:
```

Comment:
It calls `_get_device_info`, which has a different return type signature? Which one is correct?

Reply:
Fixed